### ISSUE TITLE
Drop-down Menus Not Opened Smoothly

### ISSUE THREAD

**Issue by zo-shin** [zaps166/QMPlay2#324]
In version 20.05.02 win64, all the drop-down menus can’t be opened smoothly. They get stuck for less than half a second before showing the full items.
And there are three new error logs:
initInstance: No Vulkan library available
Failed to create platform Vulkan instance
Vulkan is unable to work with QMPlay2 on this platform: Can't create Vulkan instance: ErrorInitializationFailed
Tested on Windows 7 SP1 x64. All the cases don’t exist in version 19.12.19 win64 and version 20.05.02 win32.

**Comment by zaps166** [comment]
```
initInstance: No Vulkan library available
Failed to create platform Vulkan instance
Vulkan is unable to work with QMPlay2 on this platform: Can't create Vulkan instance: ErrorInitializationFailed
```

This is OK if you don't have Vulkan drivers (old drivers and/or old hardware). It is from detection whether Vulkan works. You can go to renderer settings and click apply if OpenGL is chosen to silence this (it enforces the OpenGL).

> They get stuck for less than half a second before showing the full items.

I changed Qt to 5.14.2 for 64bit Windows build, maybe this is the problem...

I need more details.

**Comment by zo-shin** [comment]
Do I have to provide anything else?

**Comment by zaps166** [comment]
> Do I have to provide anything else?

A screenshot and screens configuration (resolution, font DPI/scale, is it a multiscreen?)

---

99% it is a new Qt bug (if doesn't happen on 32-bit which still uses Qt 5.6.3), maybe you can already find it on Qt Jira https://bugreports.qt.io/projects/QTBUG/issues/ - then you can add link here. At least I can fix it on Qt side, but I need a way to reproduce the issue.

**Comment by zo-shin** [comment]
800x600 and 1024x768, 96 DPI, single monitor
![96dpi](https://user-images.githubusercontent.com/38669548/81181448-128e0e00-8fdf-11ea-9d81-e6e48b55ddd5.gif)

**Comment by zaps166** [comment]
![settings](https://user-images.githubusercontent.com/6239090/81185520-fa53ca00-8fb1-11ea-9031-e8a74a204e93.gif)

**Comment by zo-shin** [comment]
If it's hard to reproduce, just ignore this issue. It's not a big problem, anyway.

**Comment by zo-shin** [comment]
It has been fixed in Qt 5.15.2.

**Comment by zaps166** [comment]
Ok, but now (20.12.16) I used Qt 5.14.2 again due to more important issue (#374)
